ocs_ci/utility/pull_images_for_logging_42/pull_images.py

```diff
@@ -0,0 +1,79 @@
+from ocs_ci.utility.utils import run_cmd, clone_repo
```

Comment:
@Shrivaibavi thanks for the changes, overall looks much better, one minor change, can you put both this functions in utils.py instead

Reply:
@vasukulkarni, if I understand correctly are you pointing to the function get_scripts_from_repo, pull_packages_from_quay to be moved to utils.py?

If yes, may I know why is it recommended to move the functions to utils.py. These functions are logging specific, probably others might not want to use this.

Reply:
Also do not think that anyone else will use those function so IMHO no need to move them there.

Reply:
We just discussed this in PR https://github.com/red-hat-storage/ocs-ci/issues/408,  The idea is to have related utility functions under some common name, the directory name with version `pull_images_for_logging_42` and function name `pull_images.py` still needs correction so that we follow some convention under main folders.

a) If no one is going to use it and it is just for this one test, then I recommend we just create this function under that specific test
b) If its very specific to logging test cases, we can organize it little bit better with folder and filename for
  eg: `utility/logging/utils.py`

Reply:
Done